### DISCUSSION TITLE
Would anyone prefer if `launch.bat` pulls the latest changes?

### DISCUSSION THREAD

**Discussion by winstxnhdw** [winstxnhdw/lc-hax#158] [Ideas]
I am seeing a fair bit of users that are not very comfortable with Git for version control.

**Comment by ManuelSvQ** [comment]
Yes, I need this.

**Comment by Quesoteric** [comment]
I think it would be better for you to suggest people to use github desktop instead, has a GUI option to pull latest and easy cloning. Some people might want to keep old versions too <--(like 1% of the people that use this)

**Reply by Quesoteric** [reply]
But if you go through with this I think it would be better to put it in `requirements.bat`

**Reply by winstxnhdw** [reply]
`requirements.bat` take a long time to build because of the injector though.

**Reply by winstxnhdw** [reply]
We now have `launch-dev.bat` for those who do not want to pull latest on injection.
